publish/bower.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "it-depends",
+  "version": "${version}",
+  "description": "Lightweight dependency tracking library for JavaScript",
+  "main": "it-depends.js",
+  "authors": [
+    "Sergey Gerasimov"
+  ],
+  "license": "SEE LICENSE IN license.md",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/gerich-home/it-depends.git"
+  },
+  "keywords": [
+    "dependency",
+    "tracker",
+    "dependency-tracker",
+    "dependencytracker",
+    "lightweight"
+  ],
+  "homepage": "https://github.com/it-depends-js/it-depends-bower",
+  "moduleType": [
+    "amd"
```

Comment:
I mean that `moduleType` for bower.json is `['globals', 'amd']` and for package.json is `['globals', 'amd', 'node']`, because of bower packages is only for browsers for now. Actually, I think it is not valueable part of file, just for bots maybe..